pkg/project/auth/cache.go

```diff
@@ -67,6 +67,35 @@ func subjectRecordKeyFn(obj interface{}) (string, error) {
 	return subjectRecord.subject, nil
 }
 
+type State interface {
+	State() string
+}
+
+type reflectorState struct {
+	reflectors []*cache.Reflector
+}
+
+func (rs *reflectorState) State() string {
+	state := ""
+	for i := range rs.reflectors {
```

Comment:
There's no reason to risk confusing "1", "11", "11" as the same as "11", "1", "11".  Any csv library ought to do.